package.json

```
@@ -15,7 +15,7 @@
     "lint": "eslint \"**/*.{js,ts}\"",
     "nuke": "rm -rf node_modules && npm install && lerna exec 'rm -rf node_modules' && lerna bootstrap && npm run clean",
     "upgrade": "rm -rf node_modules package-lock.json && npm install && lerna exec 'rm -rf node_modules package-lock.json' && lerna bootstrap && npm run clean",
-    "test": "lerna run test --ignore lit --ignore lit-html --ignore lit-element --ignore @lit/reactive-element --ignore @lit-labs/motion --concurrency 1 --stream",
+    "test": "lerna run test --ignore lit --ignore tests --ignore @lit-labs/motion --concurrency 1 --stream",
```

For the local test run, if we kept `tests` but ignored all the packages that `tests` runs, it seems like that might be more efficient since it's just one `wtr` invocation, although since the tunnels are the slow part of spinning up/down wtr and we don't have those on local, not sure it's a big deal? It's certainly less fiddly to do what you have here, so we don't have to curate a big ignore list of packages.

---

I do see one test in the `lit` package (and some in `labs/motion`); seems like we should not ignore those if we go with this approach.
```suggestion
    "test": "lerna run test --ignore tests --concurrency 1 --stream",
```

---

@labs/motion is currently failing on its own, I think that's why it's being ignored currently.
But adding the lit test is definitely a change i should make